pag_nodes.py

```diff
@@ -18,6 +18,20 @@
 
     BACKEND = "Forge"
 
+def get_sigmin_sigmax(model):
+    model_sampling = model.model.model_sampling
+    sigmin = model_sampling.sigma(model_sampling.timestep(model_sampling.sigma_min))
+    sigmax = model_sampling.sigma(model_sampling.timestep(model_sampling.sigma_max))

```

Comment:
Can't we just use model_sampling.sigma_min and model_sampling.sigma_max here?

Reply:
I can't test right now but as long as it works the same I see no issue